WORKSPACE

```diff
@@ -1,21 +1,11 @@
 workspace(name = "angular_material")
 
-# Load NodeJS rules. Note that this is technically not needed because
-# `rules_typescript_dependencies()` would also load the NodeJS rules, but we specifically need
-# at least v0.14.1 which includes: https://github.com/bazelbuild/rules_nodejs/pull/341
-http_archive(
-  name = "build_bazel_rules_nodejs",
```

Comment:
`tools/BUILD.bazel` uses the `nodejs_binary` rule for the sass bundling custom rule, so I think we need to keep this.

Reply:
Yeah, but `rules_typescript` also transitively loads the `rules_nodejs` version that *matches* the TypeScript rules. It's important that the TypeScript rules always get the _according_ NodeJS rules.

Reply:
It's always better to explicitly depend on the things you use directly; in general depending on transitive dependencies is an anti-pattern.

Reply:
cc @alexeagle in case he has a comment on this

Reply:
I agree with that, just not sure how it would work the best with the TypeScript rules.

Based on my understanding, if we explicitly specify a version for `rules_nodejs`, we'll also overwrite the `rules_nodejs` version that comes with the TypeScript rules. This kind of feels like it could cause unexpected behavior if we _just_ update one of the dependencies, but since we always do the updates manually, it shouldn't be a problem.

Reply:
you can supply an explicit rules_nodejs dependency, if you choose something too old, rules_typescript's ts_setup_workspace will throw an error